.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -6,12 +6,31 @@
 Fixes / closes #????
 
 
-<!-- If there is a user-visible change in coqc/coqtop/coqchk/coq_makefile behavior and testing is not prohibitively expensive: -->
-<!-- (Otherwise, remove this line.) -->
-- [ ] Added / updated test-suite
+<!-- Remove anything that doesn't apply in the following checklist. -->
+
+<!-- If there is a user-visible change and testing is not prohibitively expensive: -->
+- [ ] Added / updated **test-suite**.
+
 <!-- If this is a feature pull request / breaks compatibility: -->
-<!-- (Otherwise, remove these lines.) -->
-- [ ] Corresponding documentation was added / updated (including any warning and error messages added / removed / modified).
-- [ ] Entry added in the changelog (see https://github.com/coq/coq/tree/master/doc/changelog#unreleased-changelog for details).
-- [ ] Overlay pull requests (if this breaks 3rd party developments in CI, see
-https://github.com/coq/coq/blob/master/dev/ci/user-overlays/README.md for details)
+- [ ] Added **changelog**.
+- [ ] Added / updated **documentation**.
+  <!-- Check if the following applies, otherwise remove these lines. -->
+  - [ ] Documented any new / changed **user messages**.
+  - [ ] Updated **documented syntax** by running `make -f Makefile.make doc_gram_rsts`.
```

Comment:
There are more details, of course, but we shouldn't go overboard here.  For example, with the current doc_grammar, if someone adds a new command or tactic, the submitter will need to add it to the documentation.  We can't enable a lint check that every tactic appears in the doc until we've updated the syntax in the rest of the doc.  (Or similarly, a check that every tactic in the doc is in the code.)

Reply:
Right, but reviewers have already been checking for this manually for many years now. So IMHO, this is just implicit in the "Added / updated documentation" general checkbox.